cockatrice/src/tab_debuglog.cpp

```diff
@@ -1,35 +1,48 @@
-#include "dlg_viewlog.h"
+#include "tab_debuglog.h"
 #include "logger.h"
 
 #include <QVBoxLayout>
 #include <QPlainTextEdit>
 
-DlgViewLog::DlgViewLog(QWidget *parent)
-: QDialog(parent)
+TabDebugLog::TabDebugLog(TabSupervisor *tabSupervisor, QWidget *parent)
```

Comment:
Is there a smaller subset of TabSupervisor you could accept as an interface instead of the whole thing? I'd like to start chipping away at the tight coupling TabSupervisor has to everything.

Or is this because `Tab` requires it? We should think about how we can break that link